### ISSUE TITLE
Cannot read property 'pop' of undefined

### ISSUE THREAD

**Issue by pablohpsilva** [vercel/next.js#21459]
**What version of Next.js are you using?**

10.0.5

**What version of Node.js are you using?**

14.4.0

**What browser are you using?**

Chrome, Safari

**What operating system are you using?**

macOS

**How are you deploying your application?**

Vercel

**Describe the Bug**

I have one specific file named "index.module.css".
I'm including it in my component like
`import styles from './index.module.css'`

The file content is:
```
.wrapper {
    padding: calc(var(--gap) * 1);
    display: flex;
    flex-direction: column;
    justify-content: center;
    background: var(--theme-primary);
    margin-bottom: calc(var(--gap) * 1);
    border-radius: 5px;
    color: var(--theme-text-secondary);
}

.title {
    color: var(--theme-text-secondary);
    font-size: var(--font-medium);
}

.cta {
    display: flex;
    justify-content: flex-end;
    align-items: center;
}

.message {
    font-size: var(--font-xx-small);
    font-style: italic;
    margin-right: calc(var(--gap) * 0.5);
}
```

Whenever I run `npm run build`, this happens:
```
> [PWA] Compile server
info  - Using external babel configuration from /project/.babelrc
info  - Creating an optimized production build  
Failed to compile.

chunk f54b42984bfe4d114461fcea2710af414ac1fe74_CSS
static/css/92123ee323c1f1e794e4.css
Cannot read property 'pop' of undefined


> Build error occurred
Error: > Build failed because of webpack errors
    at /project/node_modules/next/dist/build/index.js:15:918
    at async /project/node_modules/next/dist/build/tracer.js:1:525
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ufit@0.1.0 build: `next build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ufit@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /project/.npm/_logs/2021-01-23T01_02_09_418Z-debug.log
```

If I **comment** this import, the build works.

**Expected Behavior**

To work

**To Reproduce**

Nothing special. Read the "Describe the Bug" section.

I need help.

**Comment by davidsonsns** [comment]
Did you start your project with some template?
I just made a test with the template [with-next-css](https://github.com/vercel/next.js/tree/canary/examples/with-next-css) changing the next version for the same specified above and it's working perfectly.

This same issue was reported here #10161 and was supposed to be fixed.
Could you share a small project reproducing it?

**Comment by pablohpsilva** [comment]
I've used the nextjs default template for my project.
The issue is quite weird, to be honest. If I actually use the CSS file, the build fails. If I don't use it, it works perfectly.
If I create any other CSS file, it also works fine.

That's why it is so hard to reproduce it, or even create a small project to reproduce it. I honestly don't know what to do anymore.

**Comment by pablohpsilva** [comment]
@davidsonsns 
I've found the issue (brace yourself, it is a very weird bug).

The problem was that I was using the component I mentioned previously in multiple pages, basically repeating myself.
Then, I decided to move that component into a wrapper component and import it only once. Now the problem was solved, but still, pretty weird bug. Especially because now I'm using the wrapper on multiple pages and this wrapper also has a module.css defined in it and I'm also using other components libraries for buttons and such.

This was the only way I could fix it 🥲 .

**I've made no changes on my nextjs config file.**

---

## Component & CSS module

```css
/* index.module.css */
.wrapper {
    padding: calc(var(--gap) * 1);
    display: flex;
    flex-direction: column;
    justify-content: center;
    background: var(--theme-primary);
    margin-bottom: calc(var(--gap) * 1);
    border-radius: 5px;
    color: var(--theme-text-secondary);
}

.title {
    color: var(--theme-text-secondary);
    font-size: var(--font-medium);
}

.cta {
    display: flex;
    justify-content: flex-end;
    align-items: center;
}

.message {
    font-size: var(--font-xx-small);
    font-style: italic;
    margin-right: calc(var(--gap) * 0.5);
}
```

```tsx
import { Button } from "@/components/Button";
import styles from "./index.module.css";

const BuggyComponent = () => {
  return (
    <div className={styles.wrapper}>
      <h3 className={styles.title}>text</h3>

      <span>text</span>

      <span>text</span>

      <div className={styles.cta}>
        <span className={styles.message}>
           text
        </span>

        <Button onClick={() => null}>
          text
        </Button>
      </div>
    </div>
  );
};

export default BuggyComponent

```

## Fix example of what I've done:

**Previously**
```
/pages/A.tsx
  --> import BuggyComponent.tsx

/pages/B.tsx
  --> import BuggyComponent.tsx
```

**Now**
```
/components/Wrapper
  --> import ./index.module.css // it has its own css, like the BuggyComponent.tsx
  --> import BuggyComponent.tsx

/pages/A.tsx
  --> import Wrapper.tsx

/pages/B.tsx
  --> import Wrapper.tsx
```

**Comment by pablohpsilva** [comment]
Nope... Now the problem is elsewhere... Obviously.

**Comment by davidsonsns** [comment]
Sorry for my delay!

Just tried to reproduce following your steps. So, still unable to get this error. 
Revisiting your error message I saw the message `info  - Using external babel configuration from /project/.babelrc` and as you said that you're also using a third-party library for buttons (or wherever else). Would be very very helpful if create a repository reproducing it following your same setup initial.

It would be very appreciated, with that we could redirect our efforts only to try fixing it, rather than just trying to reproduce it. 🙃

**Comment by pablohpsilva** [comment]
Hello, good morning.

I changed the webpack config and it worked. I found a way to let webpack not bitch about the styles and everything is fine now.

Thanks :)

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
